### ISSUE TITLE
Blocking bot functions do not stop bot after finishing

### ISSUE THREAD

**Issue by beamer159** [robocode-dev/tank-royale#17]
I tested this for `Bot.forward`, `Bot.back`, and `Bot.turnGunRight`. When the `run` function contains only one of these calls, I would expect the bot to perform the action and then remain still. However, the bot instead continues the motion.

For example, consider this `run` function:

```
@Override
public void run() {
    forward(100);
}
```

I would expect the bot to move forward 100 pixels and then stop. However, the bot instead moves forward 100 pixels, and then slowly creeps forward indefinitely. The problem is not that `run` is a loop. I tried this function as well:

```
@Override
public void run() {
    forward(100);
    back(100);
}
```

In this case, the bot moves forward 100 pixels, backs up 100 pixels, and then slowly creeps backwards indefinitely.

**Comment by flemming-n-larsen** [comment]
Thank you for reporting. I am wondering how I have missed this major bug.
I guess I have watched the sample bots too much which runs their logic in a loop, and hence I do not see the issue.

I will look at this ASAP.

**Comment by beamer159** [comment]
The latest update fixed this, thanks

**Comment by beamer159** [comment]
I seem to be getting this behavior again.

**Comment by flemming-n-larsen** [comment]
It seems like the recent fix I have made for #26 (`setForward(POSITIVE_INFINITE)`) also fixes this issue.
Hence, a fix should be available with a next release.
